### PR TITLE
Mi 31 tidying fn analysis

### PR DESCRIPTION
**Pull request by tommorris168**
All changes made in fn_analysis.R, so that you can see what has changed. If you are happy with it, please request changes in your review and I will then put the functions into the different files.

### CODE REVIEW

File: R/analysis_generic.R [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# # Inputting the data in long form
+# Inputting the data in long form
```
Comment (BalancedmonkeY) [inline R/analysis_generic.R:2]: Duplicate comment

File: R/analysis_generic.R [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+bugsnetdata <- function(data, metaoutcome, treatment_list){
```
Comment (BalancedmonkeY) [inline R/analysis_generic.R:3]: Would it be possible to add a line just very briefly saying what `bugsnetdata` does?

**Review by BalancedmonkeY — CHANGES_REQUESTED** [review]
Just some minor suggestions

File: R/fn_analysis.R [hunk 3]
```diff
@@ -350,9 +350,30 @@ dataform.df <- function(newData1, treat_list, CONBI) {
...
+#' @param treat_list Data frame consisting of the treatment IDs ('Number') and the treatment names ('Label').
+#' @param model "random" or "fixed"
+#' @param outcome One of "SMD", "RD", "MD", "OR". Anything else is interpreted as RR.
+#' @param CONBI "Continuous". Anything else is interpreted as binomial.
+#' @param ref Reference treatment
+#' @return List:
+#'  - 'mtcResults' = Output from gemtc::mtc.run
+#'  - 'lstx' = Vector of treatment names
+#'  - 'ntx' = Number of treatments
+#'  - 'treat_list2' = Same as @param treat_list (TM: Not sure why this is here)
```
[the remark this answers is not in the document]
Comment (tommorris168) [reply R/fn_analysis.R]: I'm guessing that used to be the case, in pre-Janion days. I've gone backwards through all the functions that deal with treat_list, and the original one is CreateTreatmentIds in DataWrangling.R, which returns treat_list as a data frame.

File: R/fn_analysis.R [hunk 4]
```diff
@@ -350,9 +350,30 @@ dataform.df <- function(newData1, treat_list, CONBI) {
...
+#'  - 'sumresults' = summary(mtcRelEffects)
+#'  - 'a' = "fixed effect" or "random effect"
+#'  - 'mtcNetwork' = Output from gemtc::mtc.network
+#'  - 'dic' = Named vector, of the statistics 'Dbar', 'pD', 'DIC', and 'data points'
+#'  - 'model' = Same as @param model
+#'  - 'outcome' = Same as @param outcome
 baye <- function(data,treat_list, model, outcome, CONBI, ref) {
   if (outcome=="SMD" | outcome=="RD") {
-  } 
+  } # Nothing returned when either of these is the outcome type. (TM: Function never called in these cases?)
```
[the remark this answers is not in the document]
Comment (tommorris168) [reply R/fn_analysis.R]: Done.

**Review by BalancedmonkeY — APPROVED** [review]

**Pull request merged**